Enhancement/2001 CLT noise abatement, SOP, North flow traffic, and tweaks

jaialin opened the pull request:
1. Added 2DME waypoints off RWYS 18L, 18C, 18R, where planes turn onto Noise Abatement heading
2. Amended south departures to fly Noise abatement headings (this also resolves conflicts alerts during parallel departures)
3. Fixed typo on STOCR arrival which prevented arrivals from being assigned another runway
4. added FedEx to FILPZ3 arrivals (to emulate flights from Memphis and Louisville)
5. Removed UPS from SIDS/STARS they don't use in real life, adjusted spawn ratio to 0.1
6. removed DLH from SIDS STARS that they don't use (they only arrive on CHSLY5 and depart on BARMY4.RDU)
7. added FastGA planes to most arrivals
8. added North flow arrivals and departures with Noise Abatement

Resolves #2001

jaialin committed:
This pull request refines KCLT.json to conform more closely to real-world procedures.

1. Added 2DME waypoints off RWYS 18L, 18C, 18R, where planes turn onto Noise Abatement heading
2. Amended south departures to fly Noise abatement headings (this also resolves conflicts alerts during parallel departures)
3. Fixed typo on STOCR arrival which prevented arrivals from being assigned another runway
4. added FedEx to FILPZ3 arrivals (to emulate flights from Memphis and Louisville)
5. Removed UPS from SIDS/STARS they don't use in real life, adjusted spawn ratio to 0.1
6. removed DLH from SIDS STARS that they don't use (they only arrive on CHSLY5 and depart on BARMY4.RDU)
7. added FastGA planes to most arrivals

MarylandFlyer commented:
Any reason for the decrease in UPS flights?

jaialin commented:
> 

Looked at the number of UPS outbounds at CLT on Flightaware.com (https://flightaware.com/live/airport/KCLT/departures?;offset=8645;order=ident;sort=ASC), 25 departures over the last two week in a list that is about 8700 departures long.

In the future I will tweak UAL's, SKW, SWA as well. Majority of the flights should be AAL, then JIA, then PDT.

MarylandFlyer commented:
> 

Gotcha, sounds good!

jaialin committed:
Removed Dash 8 from PDT airlines

PDT runs E145 exclusively now.

Removed duplicate n/fastGA

erikquinn commented:
@jaialin Is this PR ready for review then? if so, please request one from the airport-reviewers team through the menus on the right.

jaialin committed:
Merge branch 'develop' into enhancement/2001

jaialin commented:
> @jaialin Is this PR ready for review then? if so, please request one from the airport-reviewers team through the menus on the right.

It is ready for review, however, in the right sidebar under reviewers, there's nothing for me to click. Please advise!

![Screenshot 2023-01-19 192906](https://user-images.githubusercontent.com/74117877/213592078-235a23f5-197b-4eea-8c02-940f23005af1.png)

erikquinn commented:
@jaialin Ah! I see. You weren't even in the organization. I've added you now, so you should be able to add labels and request reviews and such in the future. 👍

jaialin committed:
Update kclt.json to include North ops

Departures and Arrivals for all north ops at CLT

jaialin committed:
Updated kclt.josn

Added the non-RNAV stars, with spawn rates for turboprops and twin pistons. Adjusted spawn rates of most airlines except AAL, PDT, and JIA (all the American and its partners)

jaialin committed:
Updated kclt.josn

Added the non-RNAV stars, with spawn rates for turboprops and twin pistons. Adjusted spawn rates of most airlines except AAL, PDT, and JIA (all the American and its partners)

jaialin committed:
Update kclt.json

Updated arrival patterns for AAL

jaialin committed:
Update kclt.json
